.github/workflows/build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Check for new release
         id: event_type
         env:
-          IS_NEW_RELEASE: ${{ github.event_name != 'pull_request' && contains(github.event.head_commit.message, 'Bumping version to') }}
+          IS_NEW_RELEASE: 'true'
```

Comment:
Couldn't we just remove this step and the `|| steps.event_type.outputs.is_new_release == 'true'`check all together?